CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v2.1.0dev - [date]
+## v2.1.0 - [2025-02-14]
```

Comment:
Don't we need the next release name?!

Reply:
Ah we never included it in the changelog. So I'll add it to the release note only

Reply:
We're having "Egyptian Koshari" now :)